cmds/which/which.go

```diff
@@ -65,5 +64,8 @@ func main() {
 		p = "/ubin"
 	}
 
-	which(p, os.Stdout, flag.Args())
+	if err := which(os.Stdout, strings.Split(p, ":"), flag.Args(), allPaths); err != nil {
```

Comment:
Will this break things?
current behavior
rminnich@uroot:~/go/src/github.com/u-root/elvish$ which adfa
rminnich@uroot:~/go/src/github.com/u-root/elvish$ 

no error printed.

Reply:
oh thank you for formatting that for me github.

 `rminnich@uroot:~/go/src/github.com/u-root/elvish$ which adfa
rminnich@uroot:~/go/src/github.com/u-root/elvish$ `

Reply:
On my machine:

```
hugelgupf :: which foo
foo not found                                                                                                                                                                                                                                
hugelgupf ::  
```

Reply:
rminnich@uroot:~/go/src/github.com/u-root/u-root$ /usr/bin/which foo
rminnich@uroot:~/go/src/github.com/u-root/u-root$

Reply:
yea that was zsh.